docs/introduction/getting_started.rst

```diff
@@ -40,7 +52,10 @@ First we will construct two :ref:`site_documentation` objects for the :math:`^1\
 We now have two variables, ``H_site`` and ``C_site``, which are :ref:`site_api` objects. ``H_site``
 represents a proton site with no chemical shift. ``C_site`` represents a carbon-13 site with
```

Comment:
*represents a proton site with no chemical shift*

I think it might be better to have *represents a proton site with zero(default) chemical shift*